lib/linalg/svd/SVD.h

```diff
@@ -151,34 +115,27 @@ class SVD
      *         interval.
      */
     bool
-    isNewTimeInterval() const
+    isNewSample() const
```

Comment:
Do we need this function? I'm not sure it makes sense without time intervals. It seems to be used only internally to check whether internal data has been initialized, so maybe it can be a protected function with a name like "isFirstSample"?